apps/nowcasting-app/components/side-layout.tsx

```diff
@@ -17,7 +17,7 @@ const SideLayout: React.FC<SideLayoutProps> = ({ children, className }) => {
       <Navbar />
 
       <div className="h-10 mb-auta">{children}</div>
-      <div className="relative h-full">
+      <div className="relative h-full " style={{ zIndex: "-1" }}>
```

Comment:
why zindex with minus

Reply:
because it was blocking elements behind ` <div className="h-10 mb-auta">{children}</div>`